corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -641,6 +644,25 @@ hqDefine('geospatial/js/models', [
             updateSelectedSavedPolygonParam();
         };
 
+        self.deleteSelectedPolygonFilter = function () {
+            $.ajax({
+                type: 'DELETE',
+                url: geoJSONUrl,
+                dataType: 'json',
+                data: JSON.stringify({'polygon_id': self.selectedSavedPolygonId()}),
+                contentType: "application/json; charset=utf-8",
+                success: function (ret) {
```

Comment:
Should we add another function for `error:`? Just in case anything unexpected happens during the request.

Reply:
Added in https://github.com/dimagi/commcare-hq/pull/34717/commits/67da5d7912faefcf4318e35838085f63b84b46d4